ui/App/SharedComponents/PeerBadge.svelte

```diff
@@ -0,0 +1,31 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="ts">
+  import type { User } from "ui/src/project";
+
+  import { PeerRole, PeerType } from "ui/src/project";
+  import { Badge } from "ui/DesignSystem";
+
+  export let user: User;
+</script>
+
+<style>
+  .badge {
+    margin-left: 0.5rem;
+  }
```

Comment:
In general it’s good practice to avoid positioning a component with respect to it’s context inside the component itself as it couples the component very closely to its context.

Moreover, the alignment does not seem to work properly
![Screenshot from 2021-10-20 12-08-59](https://user-images.githubusercontent.com/3919579/138074318-b95585f7-8e24-4954-ae33-af5de7609556.png)

Considering that `PeerBadge` is only used within `UserIdentity` I’d suggest to make it directly part of that component by adding an optional `badge: string` prop or `user: User` prop. This would allow us better control of the appearance in one place.